src/Controller/ResultController.php

```diff
@@ -25,4 +28,14 @@ public function index(): string
             'footprintByCategory' => $footprintByCategory
         ]);
     }
+
+    public function add(): void
```

Comment:
Maybe not the most clever idea :
- You create function add() int the controller wich is not an action
- You instanciate two ResultManager for the same action (only one should be enough)